zsh/PKGBUILD

```diff
@@ -1,39 +1,60 @@
-    'zprofile')
+        '0001-50629-do-not-use-egrep-in-tests.patch'
+        '0001-50641-use-int-main.patch'
+        '0002-50325-fix-autocompletion.patch'
+        '0003-51862-support-texinfo-7-0.patch'
+        '0004-pcre2.patch'
+        '0005-52383-avoid-incompatible-pointer-types.patch'
+        'zprofile')
 sha512sums=('d9138b7f379ad942a5f46819d2dd52d31f3a1129f2a0d1b53d4c5cd43c318b60396da6d37c57c477b8e958fb750209aca0ae93f8c9dd42ac958de006a0ff067e'
-    'SKIP'
-    '5cc6abcdcfb4f5ad7bc4a31364ca49dfd87ae03e0082d89cc2ba1f00570f6757266ee60894ad31b562408de91494c22f177b414b03cb78c31d92328686be9860'
-    'SKIP'
-    'af6a905e83807efb614a585ac1876e0a9cc8b745911b43915b06aab46757a6df6dfc64a7a60b53cc7e62e528c04aa7460e660a1de6720476030dd746af76c6e3'
-    'fa01836f40602e158fa5e802e8f2548af751c806c87a54e1761196497b2c35a68c6f8a195a5aac22c3c27e59b80b2f50bf590f124afe3ed4a9289b00033aff1f'
-    'b287e00d8de4dc4cfb1c52bb2aef1d4b191de3512baad4c91dc81e78ddc3e5bb07297f43924b022ac44ff401a348d8a9fa366e19ddc8ea1ea72df311f5ed0034')
-validpgpkeys=('F7B2754C7DE2830914661F0EA71D9A9D4BDB27B3'
-    'E96646BE08C0AF0AA0F90788A5FEEE3AC7937444'
-    '7CA7ECAAF06216B90F894146ACF8146CAE8CBBC4')
+            'SKIP'
+            '5cc6abcdcfb4f5ad7bc4a31364ca49dfd87ae03e0082d89cc2ba1f00570f6757266ee60894ad31b562408de91494c22f177b414b03cb78c31d92328686be9860'
+            'SKIP'
+            'af6a905e83807efb614a585ac1876e0a9cc8b745911b43915b06aab46757a6df6dfc64a7a60b53cc7e62e528c04aa7460e660a1de6720476030dd746af76c6e3'
+            'c237c5a282b829fd154931c4df8e8b32514695cdde5d3b3a1ae8381169c853c11138bb8dad319bfc54fe79d8446a52cc3deaf50801b7dc323c357d7988ebc1a2'
+            'fa01836f40602e158fa5e802e8f2548af751c806c87a54e1761196497b2c35a68c6f8a195a5aac22c3c27e59b80b2f50bf590f124afe3ed4a9289b00033aff1f'
+            '891577e8c3a5beb236a7044a0960a014d8a7149ce5dba7715f07b1e4d2db475f4aab7bb6cfeadaaeffe37fb889836f368467882fba1a5ac23076548185432d51'
+            '2b6fd1d077244023591e7e2995797e90b2cda7f9c8b3978521c0b77ef1c363f175b6ebb48730deb7171c695cba23c9455d71c3ddc5a55da91832274121587af5'
+            '984c804bfa4c7c9b1ddba9f18bbd29846bf6f2b3171f39bb775c11c8b1c7b5ad2c68d2922a59265c58f6e83016d6d440d64862c529d82f90cad308fd2af757ce'
+            'b287e00d8de4dc4cfb1c52bb2aef1d4b191de3512baad4c91dc81e78ddc3e5bb07297f43924b022ac44ff401a348d8a9fa366e19ddc8ea1ea72df311f5ed0034')
+
+
 
 prepare() {
     cd "${srcdir}/${pkgbase}-${pkgver}"
 
     # 50629: do not use egrep in tests
-    patch -Np1 <../0001-50629-do-not-use-egrep-in-tests.patch
-
-    # https://github.com/zsh-users/zsh/commit/3e3cfabcc74dc79d4d8717c4e5859d8d01be6c54
-    # https://www.zsh.org/mla/workers/2022/msg00652.html
-    patch -Np1 <../fix-autocompletion.patch
+    # 50641: use 'int main()' in test C-codes in configure
+    patch -Np1 < ../0001-50629-do-not-use-egrep-in-tests.patch
+    patch -Np1 < ../0001-50641-use-int-main.patch
+    
+   	# 50325: revert 38150 and fix in calling function cfp_matcher_range() instead
```

Comment:
Here too

Reply:
weird, that wasn't appearing locally, should be fixed now though